src/test/unit_tests/test_namespace_fs.js

```diff
@@ -558,6 +559,22 @@ mocha.describe('namespace_fs', function() {
             console.log('upload_object with trailing / response', inspect(upload_res));
         });
 
+        mocha.it('get empty content dir', async function() {
+            await ns_tmp.upload_object({
```

Comment:
Just make sure that you want to use the `ns_tmp` (I think we have another option).

Reply:
looks like the only difference is that ns_tmp uses bucket path that is in TMP_PATH while ns_src uses bucket path in the current directory for some reason (I guess the intention is the src directory?). besides that they are identical. ns_tmp is used in the previous tests. so might as well use it here.also using the tmp path seems more appropriate in general